apps/ensindexer/src/lib/version-info.ts

```diff
@@ -62,18 +67,43 @@ export function getPackageVersion(packageName: string) {
 }
 
 /**
- * Get complete {@link DependencyInfo} for ENSIndexer app.
+ * Search PNPM's .pnpm directory for a package
+ * PNPM stores packages like: .pnpm/@adraffy+ens-normalize@1.10.1/node_modules/@adraffy/ens-normalize
+ */
+function searchPnpmStore(pnpmDir: string, packageName: string): string | null {
+  try {
+    const entries = readdirSync(pnpmDir);
+
+    // Convert package name to PNPM's format: @scope/name -> @scope+name
+    const normalizedName = packageName.replace("/", "+");
+
+    // Find entries that match the package name
+    // They will be in format: packagename@version or @scope+packagename@version
+    for (const entry of entries) {
+      if (entry.startsWith(normalizedName + "@")) {
+        const pkgPath = join(pnpmDir, entry, "node_modules", packageName, "package.json");
+        if (existsSync(pkgPath)) {
+          const packageJson = JSON.parse(readFileSync(pkgPath, "utf8"));
+          return packageJson.version;
+        }
+      }
+    }
+  } catch (error) {
+    // Ignore errors in this helper
+  }
+
+  return null;
+}
+
+/**
+ * Get complete {@link VersionInfo} for ENSIndexer app.
  */
-export async function getDependencyInfo(): Promise<DependencyInfo> {
+export async function getVersionInfo(): Promise<VersionInfo> {
```

Comment:
```suggestion
export async function getEnsIndexerVersionInfo(): Promise<EnsIndexerVersionInfo> {
```

Goal: We should note how we're going to have nested "VersionInfo" objects for different apps in our overall architecture. Each app's "VersionInfo" should have a name that makes it explicit exactly which app's "VersionInfo" it is related to.